### DISCUSSION TITLE
DHm support through Github Actions

### DISCUSSION THREAD

**Discussion by ajilk** [manna-harbour/miryoku#340] [Q&A] [answered]
Will there be support for Colemak-DHm variation?

<img width="371" alt="image" src="https://github.com/user-attachments/assets/995f1bb1-d4bc-4ca5-96fb-63ea21dc35c7">

**Comment by ajilk** [comment]
Looks like that's just default

**Comment by manna-harbour** [comment, marked as answer]
It's the default, but the defaults will be labelled with their values in future to make it clearer.
